MyLibrary/BusinessLogic/LibraryService/IBookService.cs

```diff
@@ -0,0 +1,66 @@
+﻿// <copyright file="IBookService.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using System.Collections.Generic;
+using BusinessLogic.LibraryModel;
+
+namespace BusinessLogic.LibraryService
+{
+    /// <summary>
+    /// Interface for services connected to Book
+    /// </summary>
+    public interface IBookService
+    {
+        /// <summary>
+        /// Gets all books
+        /// </summary>
+        /// <returns>The list of books</returns>
+        IEnumerable<Book> GetAllBooks();
+
+        /// <summary>
+        /// Gets the book by identifier
+        /// </summary>
+        /// <param name="id">The identifier of book</param>
+        /// <returns>Book with specified id</returns>
+        Book GetBookById(uint id);
+
+        /// <summary>
+        /// Removes the book
+        /// </summary>
+        /// <param name="id">The identifier of book</param>
+        /// <returns>
+        /// The id of removed book or null if there isn`t book with such id
+        /// </returns>
+        uint? RemoveBook(uint id);
+
+        /// <summary>
+        /// Creates the book
+        /// </summary>
+        /// <param name="bookName">Name of the book</param>
+        /// <param name="year">The year of publishing the book</param>
+        /// <param name="authorId">The book`s author identifier</param>
+        /// <returns>The instance of type Book</returns>
+        /// <exception cref="ArgumentException">Can`t create book! Invalid author id!</exception>
+        Book CreateBook(string bookName, int year, uint? authorId = null);
+
```

Comment:
Do u actualy need dis method? As I see, it is used only in ILibrary Service and in Library service. Check it.

Reply:
Agree